R/generate_simulated_data.R

```diff
@@ -291,6 +300,47 @@ generate_simulated_data <- function(r_in_weeks = # nolint
     )
   }
 
+  log_r_site <- spatial_rt_process(
+    log_r_state_week,
+    corr_function,
+    corr_fun_params,
```

Comment:
Can you describe what this is doing by default?

Reply:
Using the default settings, this will just provide us with the uncorrelated Rt that was done before.